src/main/java/com/rtb/projectmanagementtool/user/CreateUserServlet.java

```diff
@@ -0,0 +1,66 @@
+package com.rtb.projectmanagementtool.user;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.users.*;
+import java.io.*;
+import java.util.*;
+import javax.servlet.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+
+/** Servlet that returns task data */
+@WebServlet("/create-user")
+public class CreateUserServlet extends HttpServlet {
+
+  DatastoreService datastore;
+  Cookie cookie;
+
+  public CreateUserServlet() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  // for testing
+  public CreateUserServlet(DatastoreService datastore) {
+    this.datastore = datastore;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // call auth service to generate AuthID
```

Comment:
Move into authOps as its own method.  Ideally we'd want all other pieces to not know about UserService (aka encapsulation).  This way if later on we decide to scrap UserService and replace with something else, you'd only need to touch that one file.